style.css

```diff
@@ -432,8 +464,8 @@ textarea::placeholder {
   .container-1 img {
     width: 100%;
     height: 100%;
-    margin-top: 18px;
-    margin-left: 10px;
+    margin-top: 8px;
+    margin-bottom: -5px;
   }
 
   .container-1 {
```

Comment:
- [ ] [OPTIONA] Kindly ensure that the image is responsive to all the desktop screen sizes. I have noticed that the image at desktop screen size `774px` to about `930px` is squeezed due to the fixed width. I kindly suggest to consider using responsive width to make sure that you stick to the Figma template design 😎 
- [ ] [OPTIONAL] I also suggest that the middle list item of `ul` should be responsive and matches the Figma design.
- [ ] [OPTIONAL] On the second card where the image is aligned to the right, I have noticed that the language items are lapping over each other. Please try your best to ensure that they match the Figma template design
Your design 😎 
![image](https://user-images.githubusercontent.com/99394374/236177242-bbf66ed2-03e5-4b37-a836-0ca11eb4aa10.png)

Figma design 😎 
![image](https://user-images.githubusercontent.com/99394374/236177349-fe053be5-8cc5-4dc7-a671-98f142a31715.png)